Librocket related shader issue

This bug shows up on the first 20.0 nightly. Does not appear in 19.0 final.

> Failed to load apng: (file single-player-o.png) animation duration <= 0.0f, bad data?
> Compiling new shader:
> 	libRocket UI
>    Loading built-in default shader for: rocketui-v.sdr
>    Loading built-in default shader for: rocketui-f.sdr
> Fragment shader failed to compile:
> 0(19) : error C7011: implicit cast from "int" to "bool"
> 
> OpenGL Debug: Source:OpenGL	Type:Error	ID:1282	Severity:High	Message:GL_INVALID_OPERATION error generated. Handle does not refer to the expected type of object (GL_PROGRAM_OBJECT_ARB)
> 
> Error: A shader failed to compile! Check the debug log for more information.
> File: gropenglshader.cpp
> Line: 748

I'm guessing it's related to this commit.. #2133 But I don't know for sure.

--- a/code/def_files/data/effects/rocketui-f.sdr
+++ b/code/def_files/data/effects/rocketui-f.sdr
@@ -10,7 +10,7 @@ layout (std140) uniform genericData {
 	mat4 modelViewMatrix;
 	mat4 projMatrix;
 
-	int textured;
+	bool textured;
 	int baseMapIndex;
 };
 

